app/assets/javascripts/bastion/components/bst-alert.directive.js

```diff
@@ -25,13 +20,6 @@ angular.module('Bastion.components').directive('bstAlert', ['$animate', '$timeou
         },
         link: function (scope, element, attrs) {
             scope.closeable = 'close' in attrs;
-
-            // Fade out success alerts after five seconds
-            if (scope.type === 'success') {
-                $timeout(function () {
-                    $animate.leave(element.find('.alert'), scope.close);
-                }, SUCCESS_FADEOUT);
-            }
```

Comment:
Note that this will be completely reworked using the new angular-animate, thus the deletion.  See http://projects.theforeman.org/issues/15081.